pkg/trivy/report.go

```diff
@@ -0,0 +1,114 @@
+package trivy
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type ScanReport struct {
```

Comment:
Maybe it's a good idea to look for a library here, so we don't have to maintain this part ourselves